File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int idx = 0;
        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) == ' '){
                answer += " ";
                idx = 0;
            }else{
                if(idx % 2 == 0){
                    answer += Character.toString(s.charAt(i)).toUpperCase();
                }else{
                    answer += Character.toString(s.charAt(i)).toLowerCase();
                }
                idx++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.swift
import Foundation

func solution(_ s:String) -> String {
    var result = ""
    var idx = 0

    for char in s {
        if char == " " {
            result.append(" ")
            idx = 0
        } else {
            if idx % 2 == 0 {
                result.append(char.uppercased())
            } else {
                result.append(char.lowercased())
            }
            idx += 1
        }
    }
    return result
}